Update names

samwaseda opened the pull request.

liamhuber commented:
> Is it possible that `sphinx-build` is conflicting with the repository name? @liamhuber

Not with the repository name, with the name pip is seeing. This is going to be a problem for you down the road anyhow, since `sphinx` is already both a pypi and conda package, yeah? You probably want to call this `sphinxdft` or something.

The error:

```
/home/runner/work/_temp/ee223bb6-3a02-48f6-96ea-6cd46d56cbda.sh: line 3: sphinx-build: command not found
Error: Process completed with exit code 1.
```

The real problem:

```
Run pip install --no-deps . --no-build-isolation
Processing /home/runner/work/sphinx/sphinx
  Preparing metadata (pyproject.toml): started
  Preparing metadata (pyproject.toml): finished with status 'done'
Building wheels for collected packages: sphinx
  Building wheel for sphinx (pyproject.toml): started
  Building wheel for sphinx (pyproject.toml): finished with status 'done'
  Created wheel for sphinx: filename=sphinx-0.post0.dev1-py3-none-any.whl size=12877 sha256=cc9d83e343fd7a369b554e41fa71a6939314ec2d9e3807f2145b602a357e11b9
  Stored in directory: /tmp/pip-ephem-wheel-cache-pmbk4ab7/wheels/17/11/f5/56b1497c79bc397f2dd33b05d313c246efc7fdcb350b49f94a
Successfully built sphinx
Installing collected packages: sphinx
  Attempting uninstall: sphinx
    Found existing installation: Sphinx 7.4.7
    Uninstalling Sphinx-7.4.7:
      Successfully uninstalled Sphinx-7.4.7
Successfully installed sphinx-0.post0.dev1
```

So you conda-installed `sphinx` as desired, but you can see that pip finds _your_ module named `sphinx` so it uninstalls the documentation sphinx in favour of the dft sphinx.

samwaseda commented:
Ok so I changed the name. The `phi` in `Sphinx` stands for the wave function, which is an arbitrary choice from DFT, which I replaced with the kinetic operator `T` (or `Ti` for the actual pronunciation)  and therefor named it `stinx`.  I hope you like the new name :)
